docs/change_log/release-3.2.md

```diff
@@ -93,5 +93,4 @@ The following bug fixes are included in the 3.2 release:
 * HTML tag placeholders are no longer included in  `.toc_tokens` (#899).
 * Unescape backslash-escaped characters in TOC ids (#864).
 * Refactor bold and italic logic in order to solve complex nesting issues (#792).
-* Always wrap CodeHilite code in <code> tags (#862) </code>
-* Regex for HTML placeholders simplified for speed improvement (#928).
+* Always wrap CodeHilite code in `code` tags (#862)
```

Comment:
We already released version 3.2. Therefore, release notes for a future release would not go here. Assuming we do a bug fix release, they would go in the changelog right before the 3.2.1 release (and in the same format):

https://github.com/Python-Markdown/markdown/blob/c5f1395a944e76e95250740452706f6b16830cae/docs/change_log/index.md#L6-L9

Of course the date would be left blank until we actually do a release.

Reply:
This was just reverting the last commits addition here, but I can add a comment in the changelog if that is what you are requesting. This is based off the reverting pull request I just made.

Reply:
Yeah, we should add a comment as we are fixing a bug introduced in a previous release.

Reply:
Rebased and added changelog entry.